File: DSA/Arrays/Bit-Manipulation/array-with-bit-wise-or/javascript/arrayWithBitWiseOr.js
export const arrayWithBitWiseOr = (A, B) => {
    let count = 0;
    let ans = 0;
    for (let i = 0; i < A; i++) {
        if (parseInt(B[i]) === 1) {
            count = i + 1;
        }
        ans += count;
    }
    return ans;
};

File: DSA/Arrays/Bit-Manipulation/array-with-bit-wise-or/python/arrayWithBitWiseOr.py
def arrayWithBitWiseOr(self, A, B):
        # for 0 i will the count
        # for 1 i+1

        count = 0
        ans = 0
        for i in range(A):
            if B[i] == 1:
                count = 1+i
            ans += count
        return ans